tox.ini

```diff
@@ -71,6 +71,7 @@ deps =
     django111: -r requirements/edx/django.txt
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
+    django-oauth-toolkit-master: git+https://github.com/jazzband/django-oauth-toolkit.git@bf1525e85a06929016b1fe35d863e62e58124a2f#egg=oauth2_provider
```

Comment:
Instead of `django-oauth-toolkit-master` here, we want `django20` so this dependency is used automatically when running the tests against Django 2.0.  Then create a second line prefixed with `django21` instead for use when testing with Django 2.1.  (We don't have support in the file for testing against Django 2.2 yet, so don't worry about that for now).

Reply:
Removed the `django-oauth-toolkit-master` label and added the dependency to both `django20` and `django21`.